package.json

```diff
@@ -17,23 +17,33 @@
     "dynamodb"
   ],
   "jest": {
-    "verbose": true
+    "verbose": true,
+    "testEnvironment": "node"
   },
   "scripts": {
-    "test": "jest --watch",
+    "lint": "eslint . --cache --fix",
+    "test": "jest",
```

Comment:
why not running test in watch mode by default? I find it to be the prefered way to run tests

You can add `"test:once": "jest"` if you do need to run only one test at a time